src/it/scala/io/iohk/ethereum/txExecTest/util/DumpChainApp.scala

```diff
@@ -26,6 +28,7 @@ object DumpChainApp extends App{
     val privateNetworkId = conf.getInt("networkId")
     val startBlock = conf.getInt("startBlock")
     val maxBlocks = conf.getInt("maxBlocks")
+    val secureRandom = new SecureRandom // TODO: should it be configurable here as well?
```

Comment:
Add `extends App with NodeKeyBuilder
  with SecureRandomBuilder
  with AuthHandshakerBuilder` and remove `val`s `secureRandom` and `nodeKey` and fix the compile error -
 `secureRandom` in `PeerManager` should be `authHandShaker`.